### ISSUE TITLE
Warning when analyzing observers, if line is splitted due to being too long

### ISSUE THREAD

**Issue by web-padawan** [Polymer/polymer-analyzer#656]
Just upgraded `polymer-build` to 1.2.0 which pulled in analyzer `2.0.0-alpha.40`, and there is a following warning (severity "info") when analyzing iron-ajax:
```
'_requestOptionsChanged(url, method, params.*, headers, contentType, ' +
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  'body, sync, handleAs, jsonPrefix, withCredentials, timeout, auto)'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
vendor/iron-ajax/iron-ajax.html(329,6) info [unanalyzable-polymer-expression] - 
Can only analyze databinding expressions in string literals.
```
I think this can be a common case due to code style limitations. Could analyzer recognize this?

**Comment by FredKSchott** [comment]
Brought up in https://github.com/Polymer/polymer-cli/issues/762: would it work to use template strings here?

```
`_requestOptionsChanged(url, method, params.*, headers, contentType, 
    body, sync, handleAs, jsonPrefix, withCredentials, timeout, auto)`
```

**Comment by web-padawan** [comment]
Our app is built with Polymer 1.9 and we don't use ES2015 yet, so I'd like to avoid transpilation until we switch to Polymer 2.0.

**Comment by davidmaxwaterman** [comment]
This is also an issue for 3rd-party elements, for example iron-ajax:

```
      '_requestOptionsChanged(url, method, params.*, headers, contentType, ' +
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          'body, sync, handleAs, jsonPrefix, withCredentials, timeout, auto)'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

bower_components/iron-ajax/iron-ajax.html(346,6) info [unanalyzable-polymer-expression] - Can only analyze databinding expressions in string literals.
```

It seems like it should be a simple matter to detect that they are two simple strings and just concatenate them; if they're not simple strings, then output the warning.

**Comment by web-padawan** [comment]
FYI, my example also includes code from `iron-ajax` and template literals are not feasible due to 1.x ES5 restriction.

**Comment by merlinnot** [comment]
It does not work with ES2015 template strings either.

**Comment by davidmaxwaterman** [comment]
Disappointing this is still a problem, but I worked around it by switching off eslint:

```
/* eslint-disable max-len */
... some code longer than allowed max-len ...;
/* eslint-enable max-len */
```

It would be nice if the single line version worked:
https://github.com/eslint/eslint/issues/2074

**Comment by davidmaxwaterman** [comment]
How can I get the fix for this? I'm using polymer-cli 1.6.0
